src/Core/src/Handlers/Button/ButtonHandler.Android.cs

```diff
@@ -10,20 +10,20 @@ namespace Microsoft.Maui.Handlers
 {
 	public partial class ButtonHandler : ViewHandler<IButton, MaterialButton>
 	{
-		static Thickness? DefaultPadding;
-		static ColorStateList? TransparentColorStateList;
+		// The padding value has to be done here because in the Material Components,
+		// there is a minumum size of the buttons: 88dp x 48dp
+		// So, this is calculated:
+		//   - Vertical: 6dp*2 (inset) + 8.5dp*2 (padding) + 2.5dp*2 (text magic) + 14dp (text size) = 48dp
+		//   - Horizontal: 16dp (from the styles)
+		public readonly static Thickness DefaultPadding = new Thickness(16, 8.5);
```

Comment:
I am unable to correlate the numbers in the styles with the numbers I observe. According to the source code, the values are [`16, 4`](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/button/res/values/dimens.xml#L22-L28), but this is not relevant because buttons have a minimum size which make the padding larger - and then when the text size is changed, those are not the numbers I see.

As a result, I used a native Android app and compared the differences in size when I had a padding and when I had 0, and these are the values I see that get me a button that matches the OS.